### DISCUSSION TITLE
runet-censorship-bypass в JavaScript

### DISCUSSION THREAD

**Discussion by livxbox5** [anticensority/runet-censorship-bypass#193] [Ideas]
runet-circumvention of censorship in JavaScript for Tampermonkey please do

**Comment by ilyaigpetrov** [comment]
Could you, please, clarify.

1) Do you want us to write a script for Tampermonkey that allows to circumvent censorship of some selected blocked sites without the need to install our extension?

or

2) Do you want our extension to be able to unblock some functionality (user scripts) of Tampermonkey?

**Comment by livxbox5** [comment]
1

**Comment by ilyaigpetrov** [comment]
I don't use Tampermonkey myself but I've taken a look at its permissions in the sources and there is no "proxy" permission which is required to implement the feature you requested.
 
![tampermonkey-no-proxy](https://github.com/anticensority/runet-censorship-bypass/assets/1090811/0e355528-849b-4991-8660-5016d47e19c8)

**Comment by ilyaigpetrov** [comment]
You can try discussing it with Tampermonkey developers -- maybe they can add the "proxy" permission. But still this can be not enough for a graceful implementation of your idea.
